src/components/SettingDialog.vue

```diff
@@ -529,11 +529,33 @@
               </q-card-actions>
             </q-card>
 
-            <!-- Experimental Card -->
+            <!-- Advanced Card -->
             <q-card flat class="setting-card">
               <q-card-actions>
                 <div class="text-h5">高度な設定</div>
               </q-card-actions>
+              <q-card-actions class="q-px-md q-py-none bg-surface">
+                <div>マルチエンジン機能</div>
+                <div>
+                  <q-icon name="help_outline" size="sm" class="help-hover-icon">
+                    <q-tooltip
+                      :delay="500"
+                      anchor="center left"
+                      self="center right"
+                      transition-show="jump-left"
+                      transition-hide="jump-right"
+                    >
+                      複数のVOICEVOX準拠エンジンを利用可能にする
```

Comment:
https://github.com/VOICEVOX/voicevox/blob/main/docs/UX%E3%83%BBUI%E3%83%87%E3%82%B6%E3%82%A4%E3%83%B3%E3%81%AE%E6%96%B9%E9%87%9D.md#tooltip
ここら辺と反していそうです。（元からでしたが）
```suggestion
                      マルチエンジン機能を有効にします。複数のVOICEVOX準拠エンジンが利用できるようになります。
```

Reply:
0.14時点ではまだルール無かったので買えない方が自然な気がしたので取り下げます。

Reply:
ご指摘ありがとうございます！！
メインにマージする時に変更忘れないようにしようと思います！！